src/components/thumbnail-maker/SubMenu/AlignmentMenu.tsx

```
@@ -0,0 +1,49 @@
+    <div className="flex flex-col gap-2">
+      <h3 className="text-sm font-medium">태그 정렬</h3>
+      <div className="flex gap-2">
+        <button
+          className={`p-2 rounded border ${
+            alignment === "start" ? "border border-gray-100" : "border-gray-50 "
+          }`}
+          onClick={() => handleAlignmentChange("start")}
+          title="좌측 정렬"
+        >
+          <AlignLeftIcon size={16} />
+        </button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)